sdk/servicebus/service-bus/src/util/errors.ts

```diff
@@ -110,21 +77,19 @@ export function getSenderClosedErrorMsg(
   }
   return (
     `The sender for "${entityPath}" has been closed and can no longer be used. ` +
-    `Please create a new sender using the "createSender" function on the ${clientType}.`
+    `Please create a new sender using the "createSender" function on the ServiceBusClient.`
```

Comment:
getSender

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8173/commits/7eeec6e0f8e321f5cf43d79c49c391d698e5aa2b